polyfill/lib/ecmascript.mjs

```diff
@@ -1426,7 +1417,7 @@ export const ES = ObjectAssign({}, ES2020, {
       dateTime.nanosecond,
       precision
     );
-    const timeZoneString = ES.ISOTimeZoneString(timeZone, instant);
+    const timeZoneString = timeZone === undefined ? 'Z' : ES.GetOffsetStringFor(outputTimeZone, instant);
```

Comment:
Does this make sense? If you call `toString({ timeZone: 'UTC' })` we'll get other output than calling `toString()`?

Reply:
That was the intention, the reasoning was that `Z` is the default format, but if you want the offset format then you supply a time zone. (And if the time zone you happen to supply is UTC, then you get a UTC offset of 0.)

Reply:
Okay, that sounds strange to me, but fair enough.